### DISCUSSION TITLE
Error when trying to install lunarvim

### DISCUSSION THREAD

**Discussion by JonathanGDiaz** [LunarVim/LunarVim#2802] [Q&A] [answered]
[UPDATE] I just made an alias for the lvim executeable in the .local/bin

When I run the command to launch the install script i get this error:

Verifying core plugins
/home/jonathan/.local/share/lunarvim/lvim/utils/ci/verify_plugins.sh: line 9: lvim: command not found
[ERROR]: Unable to verify plugins, make sure to manually run ':PackerSync' when starting lvim for the first time.

I get this error when installing both the stable and the rolling release.

**Comment by Kristopp** [comment]
Hello, Jonathan did u get this error fixed? I am having same issue

**Reply by JonathanGDiaz** [reply]
I didn't exactly fixed it. It seems that everything is installed but there is not executable, even though it is in the ".local/bin" folder.
When I give the file executable permission and ran it it worked, so I simply made an alias for not writing the whole path.

**Comment by rebuilt** [comment, marked as answer]
It's likely `~/.local/bin` isn't in your path.  You can verify it by running `echo $PATH` from the command line. Follow the instructions to add `~/.local/bin` to your path:  
https://lunarvim.org/docs/quick-start#add-lvim-to-path

**Reply by puang59** [reply]
Can you please update the link

**Reply by rebuilt** [reply]
https://lunarvim.org/docs/quick-start#add-lvim-to-path
